sdk/storage/storage-blob/src/BlobClient.ts

```diff
@@ -1119,6 +1269,151 @@ export class BlobClient extends StorageClient {
     });
   }
 
+  /**
+   * Set the owner, owner group or access control list for a directory.
+   *
+   * @see https://docs.microsoft.com/en-us/rest/api/storageservices/datalakestoragegen2/path/update
+   *
+   * @param {string} accessControl Sets POSIX access control rights on files and directories. The value is a comma-separated list
+   *                               of access control entries. Each access control entry (ACE) consists of a scope, a type, a user
+   *                               or group identifier, and permissions in the format "[scope:][type]:[id]:[permissions]".
+   *
+   * @param {string} accessControl
+   * @param {BlobSetAccessControlOptions} [options={}]
```

Comment:
[options] or [options={}], I find it's different with others, we need some convention.

Reply:
fixed

---
In reply to: [318944868](https://github.com/Azure/azure-sdk-for-js/pull/4874#discussion_r318944868) [](ancestors = 318944868)